tests/test_mock_command_handler.py

```diff
@@ -110,6 +110,8 @@ async def test_create_device(self) -> None:
     async def test_configure(self) -> None:
         await self.command_handler.configure(configuration=self.configuration)
         assert self.configuration == self.command_handler.configuration
+        # Make sure that all background threads of mock devices are stopped.
```

Comment:
Are you using multi-threading? If not, I suggest "tasks" instead of "threads".

Reply:
This comment was removed (see previous answer to your questions here on GitHub).